modules/programs/vscode.nix

```diff
@@ -59,6 +62,24 @@ in
         '';
       };
 
+      keybindings = mkOption {
+        type = types.listOf (types.attrsOf types.str);
```

Comment:
Wouldn't it work to have `type = types.attrsOf types.str`? Seems to me that
```
keybindings = {
  "ctrl+c" = { command = …; when = …; }
};
```
might be a bit more intuitive.

Reply:
No, because "ctrl+c" could be bound to different things in different contexts (which can be filtered by "when").

Reply:
Fair enough, I think that's a good argument to keep it as a list then.

But might be good to make it a list of submodules? Something like
```
type = types.listOf (submodule {
  key = mkOption { … };
  when = mkOption { … };
  command = mkOption { … };
});
```
so that documentation can show what should go in the list and the elements can be type checked.